internal/host/hostcommands/install_cmd.go

```diff
@@ -248,14 +248,13 @@ func (i *installCmd) getDiskUnbootableCmd(ctx context.Context, host models.Host)
 	for _, disk := range inventory.Disks {
 		isFcIscsi := strings.Contains(disk.ByPath, "-fc-") || strings.Contains(disk.ByPath, "-iscsi-")
 		if disk.Bootable && !isFcIscsi && !disk.IsInstallationMedia {
-			dev := hostutil.GetDeviceFullName(disk.Name)
-			formatCmds += fmt.Sprintf("dd if=/dev/zero of=%s bs=512 count=1 ; ", dev)
+			formatCmds += fmt.Sprintf("dd if=/dev/zero of=%s bs=512 count=1 ; ", disk.ID)
 			i.eventsHandler.AddEvent(
 				ctx,
 				host.ClusterID,
 				host.ID,
 				models.EventSeverityInfo,
-				fmt.Sprintf("Performing quick format of disk %s on host %s", dev, hostutil.GetHostnameForMsg(&host)),
+				fmt.Sprintf("%s: Performing quick format of disk %s", hostutil.GetHostnameForMsg(&host), disk.ID),
```

Comment:
Also here. The user doesn't care about disk ID's, they only care about paths.

Reply:
please review the latest code. I changed it already